### PR TITLE
remove ll feature flag, it's time!

### PR DESCRIPTION
**Pull request by bwindels**
PR to update config on develop: https://github.com/matrix-org/internal-config/pull/582
riot-web PR to fix app tests: https://github.com/vector-im/riot-web/pull/8464

### CODE REVIEW

**Comment by turt2live** [comment]
Fixes https://github.com/vector-im/riot-web/issues/7655

File: src/components/views/settings/tabs/LabsSettingsTab.js [hunk 1]
```diff
@@ -53,13 +53,6 @@ export class LabsSettingToggle extends React.Component {
     }
 
     _onChange = async (checked) => {
-        if (this.props.featureId === "feature_lazyloading") {
-            const confirmed = await this._onLazyLoadChanging(checked);
```
Comment (turt2live) [inline src/components/views/settings/tabs/LabsSettingsTab.js]: if we're dropping the homeserver check, please also delete the function above.

**Comment by turt2live** [comment]
I once again reviewed a change that wasn't in the queue - sorry :(

File: src/components/views/settings/tabs/LabsSettingsTab.js (hunk 1 above)
Comment (bwindels) [reply src/components/views/settings/tabs/LabsSettingsTab.js]: good catch

**Comment by turt2live** [comment]
also don't forget to run `npm run i18n` to get rid of all the old strings

**Comment by bwindels** [comment]
green!!!! \o/ :heavy_check_mark::heavy_check_mark::heavy_check_mark::heavy_check_mark:

**Pull request merged**